sklearn/pipeline.py

```diff
@@ -179,9 +182,12 @@ def set_output(self, *, transform=None):
             Configure output of `transform` and `fit_transform`.
 
             - `"default"`: Default output format of a transformer
-            - `"pandas"`: DataFrame output
+            - `"polars"`: Polars output
             - `None`: Transform configuration is unchanged
 
+            .. versionadded:: 1.4
+                `"polars"` option was added.
```

Comment:
Line 189 says it is *adding* `polars` as an option, but on 182 it actually replaces the `pandas` option. Is that intended?

Reply:
This was a mistake. I opened https://github.com/scikit-learn/scikit-learn/pull/27992 to fix the docstring.